lighthouse-core/audits/dobetterweb/incorrect-layers.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Audit a page to see if it does not use incorrect layers that overlap layer or huge and repaninted layer.
+ */
+
+'use strict';
+
+const Audit = require('../audit');
+const Formatter = require('../../report/formatter');
+const ONEMB = 1048576;
+const HUGE_LAYER = '1MB';
+
+
+function bytesToSize(bytes) {
+  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB'];
+  if (bytes == 0) return '0 Byte';
+  const i = parseInt(Math.floor(Math.log(bytes) / Math.log(1024)));
+  return String(Math.round(bytes / Math.pow(1024, i), 2)) + sizes[i];
```

Comment:
i like this little bytesToSize snippet. :) very clever

but afaik math.round doesn't take a 2nd arg. `toLocaleString` is always a winner. how about:

```js
var bytesStr = (bytes / Math.pow(1024, i)).toLocaleString(undefined, {maximumFractionDigits: 2});
return `${bytesStr} ${sizes[i]}`;
```